runtime-extension-model/src/main/java/org/mule/runtime/config/internal/validation/ScatterGatherRoutes.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.config.internal.validation;
+
+import static java.util.Optional.empty;
+import static java.util.Optional.of;
+import static org.mule.runtime.api.component.ComponentIdentifier.builder;
+import static org.mule.runtime.ast.api.util.ComponentAstPredicatesFactory.currentElemement;
+import static org.mule.runtime.ast.api.validation.Validation.Level.ERROR;
+import static org.mule.runtime.ast.api.validation.ValidationResultItem.create;
+import static org.mule.runtime.internal.dsl.DslConstants.CORE_PREFIX;
+
+import org.mule.runtime.api.component.ComponentIdentifier;
+import org.mule.runtime.ast.api.ArtifactAst;
+import org.mule.runtime.ast.api.ComponentAst;
+import org.mule.runtime.ast.api.validation.Validation;
+import org.mule.runtime.ast.api.validation.ValidationResultItem;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.function.Predicate;
+
+@IgnoreOnLazyInit
+public class ScatterGatherRoutes implements Validation {
+
+  private static final String SCATTER_GATHER_ELEMENT = "scatter-gather";
```

Comment:
use the one in CoreDslConstants if possible.

Reply:
CoreDslConstants is in spring-config, which is not a dependency of this module

Reply:
Ok!